docs/proposals/19-migrating-observability-stack-to-operators.md

```diff
@@ -1,23 +1,25 @@
 ---
```

Comment:
Adapt the link in https://github.com/gardener/gardener/blob/master/docs/README.md to the new file name